crm_claim/__manifest__.py

```diff
@@ -23,6 +23,7 @@
         'views/res_partner_views.xml',
         'views/crm_claim_menu.xml',
         'security/ir.model.access.csv',
+        'security/crm_claim_security.csv',
```

Comment:
:open_mouth: mistyped?? it should be `*.xml` not `*.csv`

Reply:
Oops, fixed